cli/foundation-models/system/inference/image-object-detection/image-object-detection-online-endpoint.sh

```diff
@@ -52,14 +52,19 @@ az ml online-endpoint create --name $endpoint_name $workspace_info  || {
     echo "endpoint create failed"; exit 1;
 }
 
-# deploy model from registry to endpoint in workspace
-az ml online-deployment create --file deploy-online.yaml $workspace_info --all-traffic --set \
-  endpoint_name=$endpoint_name \
-  model=azureml://registries/$registry_name/models/$model_name/versions/$model_version \
-  instance_type=$deployment_sku || {
-    echo "deployment create failed"; exit 1;
+# Deploy model from registry to endpoint in workspace
```

Comment:
Let's make the change for instance segmentation as well at the same time.

Reply:
ok

Reply:
Updated Segmentation files @rjaincc  can you please review Thanks in advance